velox/exec/HashTable.h

```diff
@@ -192,16 +192,17 @@ class BaseHashTable {
   /// decode grouping keys from 'input'. If 'ignoreNullKeys' is true, updates
   /// 'rows' to remove entries with null grouping keys. After this call, 'rows'
   /// may have no entries selected.
-  void prepareForGroupProbe(
+  virtual void prepareForGroupProbe(
       HashLookup& lookup,
       const RowVectorPtr& input,
       SelectivityVector& rows,
-      bool ignoreNullKeys,
-      int8_t spillInputStartPartitionBit);
+      int8_t spillInputStartPartitionBit = kNoSpillInputStartPartitionBit) = 0;
```

Comment:
Let's not supply default argument so people don't forget pass in the correct value

Reply:
make sense